imagery/utils/math.ts

```diff
@@ -1,6 +1,5 @@
 // Converts from degrees to radians.
 import { Point as GeoPoint, Point } from 'geojson';
-import { bearing } from '@turf/turf';
 import * as turf from '@turf/turf';
```

Comment:
remove import * as turf from '@turf/turf';
and add the actual imports like import { bearing } from '@turf/turf';